Makefile

```diff
@@ -1,10 +1,10 @@
 IMAGE_REG=quay.io
-IMAGE_ORG=integreatly
+IMAGE_ORG=croche
 IMAGE_NAME=cloud-resource-operator
 MANIFEST_NAME=cloud-resources
 NAMESPACE=cloud-resource-operator
 PREV_VERSION=0.8.2
-VERSION=0.9.0
+VERSION=1.9.0
```

Comment:
Just in case this got overlooked before merge. We should only be bumping a minor version here. The public API of CRO should not be considered stable (https://semver.org/#spec-item-4).